<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1255</width>
    <height>820</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Wizard</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QSplitter" name="splitter_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QTabWidget" name="DataTabs">
           <property name="currentIndex">
            <number>1</number>
           </property>
           <widget class="QWidget" name="SelectedFilesTab">
            <attribute name="title">
             <string>Selected Files</string>
            </attribute>
            <layout class="QGridLayout" name="gridLayout_7">
             <item row="0" column="0">
              <widget class="QTextEdit" name="SelectedFilesText">
               <property name="textInteractionFlags">
                <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="ChoicesTab">
            <attribute name="title">
             <string>Choices</string>
            </attribute>
            <layout class="QGridLayout" name="gridLayout_6">
             <item row="0" column="0">
              <widget class="QTextEdit" name="ChoicesText">
               <property name="font">
                <font>
                 <family>Consolas</family>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="textInteractionFlags">
                <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
          <widget class="QGroupBox" name="FomodWizardGroup">
           <property name="title">
            <string>Fomod Wizard</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QSplitter" name="FomodPhotoVSplitter">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <widget class="QFrame" name="frame">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                 <widget class="QLabel" name="FomodNameLabel">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>Name: Patch for animations</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="FomodMetaLabel">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>Author: Indarello, Version: 6.2, Website: Link</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="FomodOptionPhoto" name="FomodPhotoLabel">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>Photo Label</string>
                  </property>
                  <property name="scaledContents">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="">
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <item>
                 <widget class="QSplitter" name="splitter">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <widget class="QGroupBox" name="groupBox_2">
                   <property name="title">
                    <string>Instructions</string>
                   </property>
                   <layout class="QGridLayout" name="gridLayout_2">
                    <item row="0" column="0">
                     <widget class="QTextEdit" name="FomodInstructions"/>
                    </item>
                   </layout>
                  </widget>
                  <widget class="FomodOptionsPanel" name="FomodOptionsGroup">
                   <property name="title">
                    <string>Options</string>
                   </property>
                  </widget>
                 </widget>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout">
                  <item>
                   <spacer name="horizontalSpacer">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <widget class="QPushButton" name="FomodBackButton">
                    <property name="text">
                     <string>Back</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="FomodNextButton">
                    <property name="text">
                     <string>Next</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1255</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FomodOptionPhoto</class>
   <extends>QLabel</extends>
   <header>skypackages.ui.fomod</header>
  </customwidget>
  <customwidget>
   <class>FomodOptionsPanel</class>
   <extends>QGroupBox</extends>
   <header>skypackages.ui.fomod</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
